service/src/test/resources/application.yaml

```diff
@@ -101,9 +101,9 @@ application:
   kafka:
     producer:
       events:
-        topic: ${ENV:folio}.workflow.events
+        topic: ${ENV:folio}.camunda.events
 
-# use to store uploads shared over kafka
+# Used by Kafka to manage uploads.
```

Comment:
I wasn't entirely happy with the original comment, but this is even more confusing. The `event.upload.path` property is a path to an uploads directory on a mount that is to be shared between mod-workflow and mod-camunda (workflow engine) for when the event to start a workflow has a file upload. The file is stored by mod-workflow and opened by mod-camunda.